serializations.md

```diff
@@ -0,0 +1,321 @@
+# Serialization formats
+
+The serialization formats in this document describes the binary format
+used to encode the different objects in the binary format that is used
+for hashing. Other formats may be used for communication between
+nodes.
+
+## Static size object serialization
+
+#### Block header
+
+For block headers all fields sizes are statically known and can be
+constructed directly as a byte array.
+
+| Fieldname | Size (bytes) |
+| --- | --- |
+| height       | 8    |
+| prev_hash    | 32   |
+| txs_hash     | 32   |
+| root_hash    | 32   |
+| target       | 8    |
+| nonce        | 8    |
+| time         | 8    |
+| version      | 8    |
+| pow_evidence | 168  |
+
+
+#### Block
+
+The only difference between a block and its header is the list of
+transactions. The transactions are captured in the header by the
```

Comment:
... and - in a sense - the state trees.

Reply:
The state tree is represented in the header as well, and the state tree is as connected to the header as to the block. I think this reads well as is.